content/save-load-arrays.ipynb

```diff
@@ -0,0 +1,323 @@
+{
```

Comment:
<p>Might be worth removing the <code>./</code>  as I'm not sure this works on all systems (like Windows)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>good point, I'm so used to Linux now</p>